Client/Ecosia/Migration/EcosiaImport.swift

```diff
@@ -94,14 +112,15 @@ final class EcosiaImport {
                         self.migrationGroup.leave()
                     }
                 }
-
                 self.migrationGroup.leave()
             }
         }
 
         migrationGroup.notify(queue: .main) {
```

Comment:
Are we sure we don't need to worry about retain cycles, specially with espacing closures?

Reply:
Your converns made me double check I can assure you:
- denit on EcosiaImport is called after migration is done (I debugged it and set a breakpoint)
- also giving it a second thought it is actually good that those blocks retain self (EcosiaImport instance) until these blocks are called because AppDelegate is not retaining it 👍